syntax="proto2";
package baikaldb.pb;
//option cc_enable_arenas = true;

enum ReverseNodeType {
    REVERSE_NODE_NORMAL = 0;
    REVERSE_NODE_DELETE = 1;
};

//--common
message CommonReverseNode
{
    optional bytes key = 1;//must
    required ReverseNodeType flag = 2;//must
    optional float weight = 3;
};
message CommonReverseList
{
    repeated CommonReverseNode reverse_nodes = 1;//must
};
